.github/workflows/ci.yml

```
@@ -0,0 +1,30 @@
+name: Continuous Integration
+on: [push]
```

```suggestion
on: [push, pull_request]
```

in order to trigger the CI build on PR from forks